libs/core/src/main/java/org/opensearch/core/rest/RestStatus.java

```diff
@@ -559,4 +567,30 @@ public static RestStatus status(int successfulShards, int totalShards, ShardOper
     public static RestStatus fromCode(int code) {
         return CODE_TO_STATUS.get(code);
     }
+
+    public static boolean isValidRestCode(final int code) {
+        return fromCode(code) != null;
+    }
+
+    /**
+     * Get category class of a rest status code.
+     *
+     * @return String representing class category of the concrete rest status code
+     */
+    public String getClassName() {
```

Comment:
Can we rename to getStatusFamily (similar to JAX-RS)?

Reply:
Sure, makes sense.